mne/stats/cluster_level.py

```diff
@@ -1331,6 +1369,16 @@ def spatio_temporal_cluster_test(
     H0 : array, shape (n_permutations,)
         Max cluster level stats observed under permutation.
 
+    Notes
+    -----
+    For computing a ``threshold`` based on a p-value, use the conversion
+    from :meth:`scipy.stats.f.ppf`::
+
+        pval = 0.001  # arbitrary
+        dfn = n_conditions - 1  # degrees of freedom numerator
+        dfd = n_observations - n_conditions  # degrees of freedom denominator
+        thresh = scipy.stats.f.ppf(1 - pval, dfn=dfn, dfd=dfd)
```

Comment:
same comment as above: not DRY

Reply:
also in a9197fde6f277e8758a288389c1ba727fb781720